lib/rbovirt.rb

```diff
@@ -131,10 +133,11 @@ def http_put(suburl, body, headers={})
       end
     end
 
-    def http_delete(suburl)
+    def http_delete(suburl, body, headers={})
```

Comment:
By changing the method to 
```ruby
def http_delete(suburl, body=nil, headers={})
```
You will not need to use nil each time you call the method.